base58/README.md

```diff
@@ -16,6 +16,17 @@ crate differs from them because:
    addresses. It may be appropriate in cases where performance is more important than safety.
 
 
+## Crate name
```

Comment:
I think this is redundant, the section above already mentions other crates. Just add `base58check` to the list.

Reply:
I don't know what I was doing yesterday. R-wrote the readme section, please note @apoelstra I realised that `base58checksum` depends on `base58` (and `sha1`) so I mentioned this and removed some of the text that you previously suggested.